app/hooks/server_actions.ts

```diff
@@ -230,12 +232,34 @@ export async function isTableExisting(table: string) {
     }
 }
 
-export async function getPendingPlayers(table: string) {
-    const backendUrl = process.env.NEXT_PUBLIC_API_URL;
-    if (!backendUrl) {
-        throw new Error('Backend API URL is not defined');
+export async function isTableOwner(table: string, clientUuid: string) {
+    isBackendUrlValid();
+
+    if (!table || !clientUuid) {
+        throw new Error('Table/clientUuid is not defined');
     }
 
+    try {
+        const response = await fetch(`${backendUrl}/is-table-owner`, {
+            method: 'POST',
+            headers: {
```

Comment:
You need to add 

credentials: "include" 

Here. This is what allows the user to pass the cookies to the backend.